src/main/java/org/openrewrite/gradle/RecipeMarketplaceCsvValidateCompletenessTask.java

```diff
@@ -89,21 +93,27 @@ void validate() {
 
         // Load environment from JAR only (not dependencies)
         Environment jarEnvironment = Environment.builder()
-                .load(new ClasspathScanningLoader(new Properties(), new RecipeClassLoader(recipeJarPath, Collections.emptyList())))
+                .load(new ClasspathScanningLoader(new Properties(), new RecipeClassLoader(recipeJarPath, emptyList())))
```

Comment:
Still exploring the absence of the Refaster recipes when loading from the environment; possibly this loader and the empty classpath have something to do with it.